src/core/client/admin/components/StoryInfoDrawer/RescrapeStory.tsx

```diff
@@ -0,0 +1,42 @@
+import { Localized } from "@fluent/react/compat";
+import RescrapeStoryMutation from "coral-admin/routes/Stories/StoryActions/RescrapeStoryMutation";
+import { useMutation } from "coral-framework/lib/relay";
+import { Button } from "coral-ui/components/v2";
+import React, { FunctionComponent, useCallback, useState } from "react";
+
+import styles from "./RescrapeStory.css";
+
+export interface Props {
+  storyID: string;
+}
+
+const ScrapeStory: FunctionComponent<Props> = ({ storyID }) => {
+  const [triggered, setTriggered] = useState(false);
+  const triggerRescrape = useMutation(RescrapeStoryMutation);
+  const rescrape = useCallback(async () => {
+    if (triggered) {
+      return;
+    }
+
+    await triggerRescrape({ id: storyID });
+    setTriggered(true);
+  }, [storyID, triggerRescrape, triggered]);
+
+  return (
+    <Button
+      className={styles.button}
+      type="button"
+      onClick={rescrape}
+      variant="outlined"
+      color="mono"
+    >
+      {triggered ? (
+        <Localized id="storyInfoDrawer-rescrapeTriggered">Triggered</Localized>
+      ) : (
```

Comment:
When the rescrape is triggered, does it make sense to also disable this button? I don't know if we want them to be able to click the button if it's already re-scraping.